docs/nodes/validator.md

```
@@ -0,0 +1,127 @@
+   }
+   ```
+
+   Max-change-rate, set the initial self-delegation `amount`, and must replace the key field with your own validator `key` from earlier.
+
+  :::warning
+  When setting commission parameters, the `commission-max-change-rate` is measured **in percentage points** of the `commission-rate`.  
+  For example: changing from `1%` to `2%` represents a `100%` relative increase,  
+  but the `commission-max-change-rate` value would be `0.01` (1 percentage point).
+  :::
+
+
```

markdownlint-cli2 (0.17.2)

Multiple consecutive blank lines
Expected: 1; Actual: 2

(MD012, no-multiple-blanks)